benefits/eligibility/templates/eligibility/start.html

```diff
@@ -17,7 +17,7 @@ <h2 class="media-title p-sm">{{ start_sub_headline }}</h2>
         {% block media_list %}
           {% if media|length_is:"0" %}
           {% else %}
-            <ul class="media-list mt-0 p-0 d-flex justify-content-center flex-column">
+            <ul class="media-list mx-0 p-0 d-flex justify-content-center flex-column">
```

Comment:
Maybe this doesn't matter since all this media list HTML is going to be replaced by the `media-list` include in #1013... but shouldn't this match what you have in `includes/media-list.html`?

Reply:
Yeah - I think the logic should both be, "Zero margins, zero padding left/right" - so gonna change both to `m-0 px-0`